grafana/Dockerfile

```diff
@@ -13,12 +13,9 @@ ENV GF_ANALYTICS_REPORTING_ENABLED=false \
     GF_USERS_DEFAULT_LANGUAGE=detect \
     GF_DASHBOARDS_DEFAULT_HOME_DASHBOARD_PATH=/dashboards_internal/home.json \
     DATABASE_PORT=5432 \
-    DATABASE_SSL_MODE=disable
+    DATABASE_SSL_MODE=disable \
+    GF_DATE_FORMATS_USE_BROWSER_LOCALE=true
```

Comment:
this has been fixed via https://github.com/grafana/grafana/pull/97971

Reply:
i left the config setting disabled in nixOS until a stable version of nixOS including Grafana 11.5.0 is available

Reply:
Perfect, ty!